modules/frontend/tag_sharder.go

```diff
@@ -296,14 +296,13 @@ func (s searchTagSharder) buildBackendRequests(ctx context.Context, tenantID str
 
 		blockID := m.BlockID.String()
 		for startPage := 0; startPage < int(m.TotalRecords); startPage += pages {
-			subR := parent.Clone(ctx)
-			subR, err := searchReq.buildTagSearchBlockRequest(subR, blockID, startPage, pages, m)
+			pipelineR, err := cloneChildRequest(parent, tenantID, func(r *http.Request) (*http.Request, error) {
+				return searchReq.buildTagSearchBlockRequest(r, blockID, startPage, pages, m)
+			})
 			if err != nil {
 				errFn(err)
-				return
+				continue
```

Comment:
What's the implication of this? Before we failed fast

Reply:
before we stopped creating jobs if this err hit and only issued the first N jobs. now we just skip one job. this failure should basically never happen, but we log it anyway.

we should do a better job of propagating this error back to the user if it were to ever happen